app/index.html

```diff
@@ -96,6 +96,7 @@ <h2>Welcome to the Developer Tools</h2>
                             <button type="button" class="regionButton" value="mypurecloud.com.au">APAC South (mypurecloud.com.au)</button>
                             <button type="button" class="regionButton" value="apne2.pure.cloud">APAC NorthEast (Seoul) (apne2.pure.cloud)</button>
                             <button type="button" class="regionButton" value="mypurecloud.jp">APAC NorthEast (Tokyo) (mypurecloud.jp)</button>
+                            <button type="button" class="regionButton" value="aps1.pure.cloud">Mumbai (aps1.pure.cloud)</button>
```

Comment:
The name should be `APAC Southeast (Mumbai) (aps1.pure.cloud)` for consistency with the other names. Also fix the other APAC names to use `Northeast`; the E should not be capitalized.